app/grandchallenge/annotations/models.py

```diff
@@ -244,3 +244,80 @@ class ETDRSGridAnnotation(AbstractImageAnnotationModel):
 
     class Meta(AbstractImageAnnotationModel.Meta):
         unique_together = ("image", "grader", "created")
+
+
+class ImageQualityAnnotation(AbstractImageAnnotationModel):
+    """
+    Model to annotate quality of an image
+    """
+
+    QUALITY_00 = "00"
```

Comment:
Are these codes used elsewhere? Maybe make this more verbose for us developers, eg

```
QUALITY_UNGRADABLE_BAD_PHOTO = "up"
QUALITY_UNGRADABLE_CATARACT = "uc"
QUALITY_UNGRADABLE_MYDRIASIS = "um"
QUALITY_FAIR_BAD_PHOTO = "fp"
...
```

And/or split these up into 2 fields? The quality reason can then be ignored for "good"